syft/execution/communication.py

```diff
@@ -28,21 +28,31 @@ def __init__(
         super().__init__()
 
         self.obj_id = obj_id
+        if name in ["move", "remote_send", "remote_get", "get", "share", "value", "share_"]:
```

Comment:
What is value? Does it really represent a communication?

Reply:
I wondered the same thing, but it isn't a computation. I think it might be equivalent to `get` but without removing the remote instance? Not sure.

Reply:
took a closer look, it appears value is currently broken and not being used very much at all... (we could potentially remove it)